### PR TITLE
CAS-10849 setImageInfo() must be called for changes to persist

### PR DESCRIPTION
**Pull request by dmehring**

### CODE REVIEW

**Comment by gervandiepen** [comment]
Wouldn't it be easier and better to do:

+    ImageInfo rwii;
+    ndone += rwii.setInfoSplitBeamSet (ndone, info, img.shape(),
                                                     img.coordinates(),
                                                     latticeConcat_p.axis());
+    img.setImageInfo(rwii);

**Comment by dmehring** [comment]
Done.

File: images/Images/ImageConcat.tcc [hunk 1]
```diff
@@ -259,9 +259,11 @@ Bool ImageConcat<T>::setImageInfo (const ImageInfo& info)
   for (uInt i=0; i<latticeConcat_p.nlattices(); ++i) {
     ImageInterface<T>& img =
       dynamic_cast<ImageInterface<T>&>(*(latticeConcat_p.lattice(i)));
-    ndone += img.rwImageInfo().setInfoSplitBeamSet (ndone, info, img.shape(), 
+    ImageInfo& rwii = img.rwImageInfo();
```
Comment (gervandiepen) [inline images/Images/ImageConcat.tcc]: I think it is better to do
    ImageInfo rwii;
which works fine. It is somewhat dirty to change the internal ImageInfo object and set it to itself.

**Comment by gervandiepen** [comment]
See the 'Files changed' for a slight comment.

File: images/Images/ImageConcat.tcc (hunk 1 above)
Comment (dmehring) [reply images/Images/ImageConcat.tcc]: ok, then there is no point in getting the rw version. one can just make a copy of the read-only version if it is going to be copied anyway.

**Comment by dmehring** [comment]
Done.

**Comment by gervandiepen** [comment]
In fact 'ImageInfo rwii;' could be used instead of getting it from the imge, b ut it makes no difference. So it's fine now.

**Pull request merged**